chunky/build.gradle

```diff
@@ -8,6 +8,7 @@ archivesBaseName = 'chunky-core'
 dependencies {
   compile 'it.unimi.dsi:fastutil:8.4.4'
   compile 'org.apache.commons:commons-math3:3.2'
+  compile 'org.apache.commons:commons-collections4:4.4'
```

Comment:
Looks fine to me. 🤔 They don't have the primitive maps that we take from fastutil, right?

Reply:
No, they don't. 
One thing I forgot to say is that google guava provides the weak map as well but between the 2, I choose the smaller dependency. If someday guava is added (and nothing else from commons collection is used), this could be removed